tensorflow/python/lib/io/tf_record_test.py

```diff
@@ -447,6 +448,18 @@ def testWriteAndRead(self):
     actual = list(tf_record.tf_record_iterator(self._fn, self._options))
     self.assertListEqual(actual, records)
 
+  def testFlush(self):
+    records = list(map(self._Record, range(self._num_records)))
+    def childProcess(writer, rs):
```

Comment:
Function name "childProcess" doesn't conform to u'^(?:(?P<exempt>setUp|tearDown|setUpModule|tearDownModule)|(?P<camel_case>_?[A-Z][a-zA-Z0-9]*)|(?P<snake_case>_?[a-z][a-z0-9_]*))$' pattern [invalid-name]

Reply:
Does this mean that I should change `childProcess` to `ChildProcess` or `child_process`?

Reply:
Just add a # pylint: disable=invalid-name